features/integration/compile.feature

```diff
@@ -0,0 +1,13 @@
+@compile
+Feature: Compile
+  Background:
+    Given an algod v2 client
+
+  Scenario Outline: Compile programs
```

Comment:
These could probably be rewritten slightly so that the compile step only needs to be implemented once. Maybe something like this (4 steps instead of 6, per SDK)

```
When I compile a teal program <program>
Then it is compiled with <status> and <result> and <hash>

...

Given I compile a teal program <program>
When I dry run it
Then I get execution result <result>
```

Reply:
In this case one moving part is added to dryrun test. My intention was in testing one REST API endpoint in these scenario. I plan to add couple more tests and have you idea in mind.

Reply:
Rewrote to
```
    When I dryrun a <kind> program <program>
    Then I get execution result <result>
```
Dryrun can accept both source and binary so `I compile a teal program` is exceed step.